commons/ava_services/gecko_service_availability_checker_core.go

```diff
@@ -18,20 +24,15 @@ func (g GeckoServiceAvailabilityCheckerCore) IsServiceUp(toCheck services.Servic
 	client := gecko_client.NewGeckoClient(jsonRpcSocket.GetIpAddr(), jsonRpcSocket.GetPort())
 	healthInfo, err := client.HealthApi().GetLiveness()
 	if err != nil {
-		logrus.Trace(stacktrace.Propagate(err, "Error occurred in getting liveness info"))
+		// TODO increase log level when InitialDelay is set to a conservative value
```

Comment:
I added this in when planning to use an InitialDelay. The idea was that after the InitialDelay all of the services should be up, so if this ever errors it would indicate an error that the user would want to know about as opposed to the current setup where since it's just querying repeatedly, then trace would be the appropriate log level. Removing it since there won't be an initial delay.